File: go/wam/5762.go
const WamChannelMembershipActionEvent = 5762 // channel: regular

type ChannelMembershipActionEventEvent struct {
	ActionResult CHANNELMEMBERSHIPACTIONRESULT `wam:"action_result"` // field 1
	Cid string `wam:"cid"` // field 2
	UnifiedSessionId string `wam:"unified_session_id"` // field 3
	UpdatesTabSessionId int64 `wam:"updates_tab_session_id"` // field 4
}

File: rust/wam/5762.rs
pub const WAM_CHANNEL_MEMBERSHIP_ACTION_EVENT: u32 = 5762;

#[derive(Debug, Default)]
pub struct ChannelMembershipActionEventEvent {
    /// field 1, wire `action_result`
    pub action_result: Option<CHANNELMEMBERSHIPACTIONRESULT>,
    /// field 2, wire `cid`
    pub cid: Option<String>,
    /// field 3, wire `unified_session_id`
    pub unified_session_id: Option<String>,
    /// field 4, wire `updates_tab_session_id`
    pub updates_tab_session_id: Option<i64>,
}